setup.py

```diff
@@ -4,10 +4,10 @@
     long_description = fh.read()
 
 setup(
-    name="AutoDH",
+    name="osaro-AutoDH",
     version="0.2",
```

Comment:
use versioner here. Don't want to just hard code versions. See this example: https://github.com/OsaroAI/osaro-default-python-template/blob/main/setup.py#L12